raiden/tests/utils/geth.py

```diff
@@ -129,34 +140,36 @@ def geth_create_account(datadir, privkey):
     assert create.returncode == 0
 
 
-def geth_bare_genesis(genesis_path, private_keys, random_marker):
+def geth_generate_poa_genesis(
+        genesis_path,
+        accounts_addresses,
+        seal_address,
+        random_marker,
+):
     """Writes a bare genesis to `genesis_path`.
 
     Args:
         genesis_path (str): the path in which the genesis block is written.
```

Comment:
Would be nice to move the type annotations out of the docstring into the function signature.